### PR TITLE
Supporting till version 6.6.2.

### PR DESCRIPTION
**Pull request by juremedvesek**
Updated linux containers to test, and windows msi-s. Is it ok to also update 6.2.5 to 6.2.7?
Also docker images for linux must still be pushed to quay repository.

Test were executed. Specific change was inside integration tests where i had comented 'pull=true', since i had to use local packages,
because I did not pushed images to quay yet.

### CODE REVIEW

**Review by tadeboro — CHANGES_REQUESTED** [review]
I added some inline comments about the changes, but there is one more thing I would like to see addressed. Can we split this PR into two PRs: one for updating windows versions, and another one for updating docker images and CI config? Thanks!

File: docker/sensu-6.2.7.docker [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM sensu/sensu:6.2.7
+RUN apk update \
+    && apk add --no-cache python3 py3-bcrypt py3-six py3-cffi bash
+CMD [ \
+  "sensu-backend", "start", \
+  "--state-dir", "/var/lib/sensu/sensu-backend", \
+  "--log-level", "debug" \
+]
```
[the remark this answers is not in the document]
Comment (juremedvesek) [reply docker/sensu-6.2.7.docker:8]: Ok will split

File: docker/sensu-6.4.3.docker [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM sensu/sensu:6.4.3
+RUN apk update \
+    && apk add --no-cache python3 py3-bcrypt py3-six py3-cffi bash
+CMD [ \
+  "sensu-backend", "start", \
+  "--state-dir", "/var/lib/sensu/sensu-backend", \
+  "--log-level", "debug" \
+]
```
[the remark this answers is not in the document]
Comment (juremedvesek) [reply docker/sensu-6.4.3.docker:8]: Agreed - my bad

File: roles/install/vars/Windows.yml [hunk 3]
```diff
@@ -161,10 +161,64 @@ _msi_lookup:
...
+    product_codes:
+      x64: '{F9FEE5BE-91EE-45BC-AED4-B647A8E4AD08}'
+      x86: '{C4BDBBA0-4EBE-471F-BF7E-704C24C33197}'
+    version: 6.6.1
+  6.6.2: &id001
+    build: 5538
+    product_codes:
+      x64: '{C9AB16C4-34E5-4237-A577-8941A96AFDF7}'
+      x86: '{662D43FF-8EE5-48CB-A2FD-9F2FE1C18CBB}'
+    version: 6.6.2
```
[the remark this answers is not in the document]
Comment (juremedvesek) [reply roles/install/vars/Windows.yml:223]: Will probably create separate PR - need to check what "update_windows_version" produces, and if we need to add some "skip" logic there for old versions.

**Comment by tadeboro** [comment]
Closing as this was implemented in #307 and #306

**Pull request closed without merging**